op-challenger/game/keccak/matrix/matrix.go

```diff
@@ -18,7 +19,10 @@ type StateMatrix struct {
 // LeafSize is the size in bytes required for leaf data.
 const LeafSize = 136
```

Comment:
So in the pr description, it says:
> Leaf - exactly 136 bytes of data including keccak padding for the last leaf, with an index and state comment. This is the value that's hashed and added to the merkle tree

To me it'd make sense to move this into the keccak types and call it `BlockSize`. It can then be used as the `Leaf.Input` field which is `[136]byte` and then also be used inside the matrix since it has "leaf"'s with size `BlockSize` except for the last leaf. Does that sound correct?

Reply:
Yeah I think that makes a lot of sense.